src/components/ui/certificate.tsx

```
@@ -103,52 +141,56 @@ const Certificate: React.FC<CertificateProps> = ({ certificates = [] }) => {
-    };
-
-    const formatFileSize = (bytes: number): string => {
-        if (bytes === 0) return '0Bytes';
-        
-        const k = 1024;
-        const sizes = ['Bytes', 'KB', 'MB', 'GB'];
-        const i = Math.floor(Math.log(bytes) / Math.log(k));
-        
-        return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + sizes[i];
+        try {
+            if (editingCertification && editingCertification.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)